benchmark_automation/benchmark_automation.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
```

Comment:
I am not sure what happened here.  Our files start with a BOM when they contain non-ASCII characters.  Did we lose the BOM?

Reply:
Done.